docs/source/connecting/connecting.md

```diff
@@ -34,6 +35,11 @@ async fn main() -> Result<(), Box<dyn Error>> {
 After successfully connecting to some specified node the driver will fetch topology information about
 other nodes in this cluster and connect to them as well.
 
+You can also set the `cluster_topology_refresh_interval` to a non-zero value to periodically refresh the
+cluster topology. This is useful when you do not have unexpected amount of traffic or when you
+have an extra traffic causing topology to change frequently. The default value is 60,
+which means that the driver will refresh the topology every 60 seconds.
+
```

Comment:
IMO, the way the first sentence is worded ("You can also set the `cluster_topology_refresh_interval` to a non-zero value to periodically refresh the cluster topology") implicitly suggests that the driver does not refresh the cluster topology if you don't set this parameter. This is clarified at the end of the paragraph, but somebody that just skims over the docmentation and reads the beginnings of the paragraphs might miss it.

Personally, I'd structure it like this:

- First mention that the driver periodically refreshes cluster topology,
- Then mention that the interval is 60s by default, but can be changed via `cluster_topology_refresh_interval`.